tools/flaky_test_monitor/level1/process_summary1_results.go

```diff
@@ -147,32 +145,80 @@ func processTestRunLineByLine(scanner *bufio.Scanner) map[string][]*common.Level
 	return testResultMap
 }
 
-func finalizeLevel1Summary(testResultMap map[string][]*common.Level1TestResult) common.Level1Summary {
+func parseSkipReason(output []string) (unittest.SkipReason, bool) {
+	for i := len(output) - 2; i >= 0; i-- {
```

Comment:
can you add a comment on why we're starting not from the last output element? Is this assuming that the last output was a pass?

Reply:
Sure, I'll add a comment. It's just because in most cases the line containing the skip reason is the last line of output (but not always).

Reply:
wait, but isn't this starting from the second last line?

Reply:
Yes, the actual last line is output by go. The second last line is the last user outputted line